specification/kubernetesconfiguration/Kubernetesconfiguration.Management/routes.tsp

```diff
@@ -0,0 +1,33 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ResourceManager;
+
+namespace Microsoft.KubernetesConfiguration;
+
+model Operation is ProxyResource<OperationsParameter> {
```

Comment:
This resource name `Operation` will collide with RP's operation list's response model. Can we rename it?